### PR TITLE
Removed delay in Panagram.prototype.end

### PR DESCRIPTION
**Pull request by SilverTactic**

### CODE REVIEW

File: chat-plugins/panagrams.js [hunk 1]
```diff
@@ -70,12 +70,7 @@ var Panagram = (function () {
 	};
 	Panagram.prototype.end = function (forced) {
 		if (forced) this.room.add('|html|The game of panagram has been forcibly ended. The answer was <b>' + this.answer.species + '</b>.');
-		if (this.sessions > 1 && !forced) {
-			setTimeout(function () {
-				pGames[this.room.id] = new Panagram(this.room, this.sessions - 1);
-				this.room.update();
-			}.bind(this), 500); //half a second
-		}
+		if (this.sessions > 1 && !forced) pGames[this.room.id] = new Panagram(this.room, this.sessions - 1);
```
Comment (panpawn) [inline chat-plugins/panagrams.js]: Why are we no longer using ``this.room.update()`` here to refresh the room?

**Comment by SilverTactic** [comment]
Because it's done automatically. You'd have to manually do it when you set a timeout, though.

**Comment by panpawn** [comment]
Where is it done automatically?

**Comment by SilverTactic** [comment]
Umm... idek how to answer that question, lol. Try using something like <code>/eval this.add('#SwegPancakes69');</code> in any room

**Comment by panpawn** [comment]
That's only "automatically updating" because it's also sending other parts from the eval, which is what I was saying for why this needs ``this.room.update()`` here.

Otherwise, this will not automatically update until someone talks or something which could then provide an unfair advantage as well as confusion.

**Comment by SilverTactic** [comment]
Hmm alright, I'll add it back in.

**Comment by panpawn** [comment]
Remember we had tested this before with panagrams on the test server by having it ``add`` to another room, and we both saw how it doesn't automatically update unless you tell it to.

**Comment by SilverTactic** [comment]
Right, okay.
(That may or may not be because the room we added to wasn't the room we were in)

**Comment by panpawn** [comment]
(It was because the server didn't add anything else to that room until we talked :-1:)

File: chat-plugins/panagrams.js [hunk 2]
```diff
@@ -106,7 +103,6 @@ exports.commands = {
 		if (!this.can('ban', null, room)) return this.errorReply("You must be ranked @ or higher to start a game of panagram in this room.");
 		if (room.id !== 'gamechamber') return this.sendReply('|html|You can only start a game of Panagram in the <button name = "send" value = "/join gamechamber">Game Chamber</button>');
 		if (!target || isNaN(target)) return this.errorReply("Usage: /panagram [number of sessions]");
-		if (target < 1) return this.errorReply("You cannot set the number of sessions to anything less than 1.");
```
Comment (panpawn) [inline chat-plugins/panagrams.js]: Why are we removing this check, @SilverTactic ?

Comment (panpawn) [inline chat-plugins/panagrams.js]: Oh, nevermind

**Pull request merged**